api/managementpb/ia/rules.proto

```diff
@@ -62,30 +62,42 @@ message RuleParam {
 
 // Rule represents Alert Rule.
 message Rule {
-  // Template used to create that rule.
-  Template template = 1;
   // Rule ID.
-  string rule_id = 2;
+  string rule_id = 1;
   // True if that rule is disabled.
-  bool disabled = 3;
+  bool disabled = 2;
   // Rule human-readable summary.
-  string summary = 4;
-  // Rule parameters.
-  repeated RuleParam params = 5;
-  // Rule duration.
-  google.protobuf.Duration for = 6;
-  // Rule severity.
-  management.Severity severity = 7;
+  string summary = 3;
```

Comment:
Memory used by MongoDB connections

Reply:
these are completely breaking changes, but probably we can proceed. Should we inform our clients about that?

Reply:
That is beta API, we can break it.